src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,14 +133,19 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, CURRENT_ZONE_OFFSET);
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
+        try {
+            return Optional.of(LocalDate.parse(date, DATE_TIME_FORMATTER_TO_PARSE_DATE));
```

Comment:
probably we have ready - made formatter for this case ?

Reply:
agree, it was not easy to find it because there are no explanations in DateTimeFormatter class